tasks.yaml

```diff
@@ -7,6 +7,14 @@ variables:
     default: ""
   - name: NGINX_EXTRA_PORTS
     default: "[]"
+  - name: COREDNS_OVERRIDES
```

Comment:
Do we need this defined here? We should be able to use the default values in the `chart/values.yaml`?

Reply:
I added this as a variable in case someone executes this as a task, and wants to supply their own CoreDNS overrides via the task

Reply:
Circling back to this after the update I still don't think we should be should be passing this variable in through the task.  Aside from just avoiding duplicating the core-dns config from `values.yaml` this task is used in the basic test we do in CI.   I'd prefer that test operated on the vanilla configuration.  I realize there was some prior example of `NGINX_EXTRA_PORTS` which did a similar thing.

Unless folks are actually consuming these tasks or there are use cases I am not aware of I would say we should just drop the the task variables that get passed into the `zarf package deploy`.  

Maybe @mjnagel or @Racer159 have other thoughts?

Reply:
@rjferguson21 I have no issues with removing this from the tasks.yaml. I was using the tasks to run my tests, but downstream users don't need this tbh

cc: @mjnagel @Racer159

Reply:
I'd prefer removing for now, agreed with Rob's comments that this is generally just used for CI testing of the package and adds extra "baggage" to duplicate the values default here.

Reply:
Removed!